store/v2/commitment/iavl/tree.go

```diff
@@ -169,6 +183,9 @@ func (t *IavlTree) PausePruning(pause bool) {
 
 // Export exports the tree exporter at the given version.
 func (t *IavlTree) Export(version uint64) (commitment.Exporter, error) {
+	if version < t.initialVersion {
```

Comment:
Why not return an error here?  Is there ever a case where calling this with this invalid version is valid?